Documentation/ApiOverview/CommandControllers/CustomCommands.rst

```diff
@@ -0,0 +1,44 @@
+:navigation-title: Custom Commands
+
+..  include:: /Includes.rst.txt
+..  _writing-custom-commands:
+
+=======================
+Writing custom commands
+=======================
+
+TYPO3 uses the Symfony Console component to define and execute command-line
```

Comment:
Limk to symfony docs maybe?

Reply:
This file will be improved in a follow up, for the time beeing I only moved the content from the introduction page here. See also the contained todo